yarn.config.cjs

```diff
@@ -231,25 +230,29 @@ function enforceExports({Yarn}) {
       let exportsImport = workspace.manifest?.exports?.import;
       if (workspace.manifest.exports?.['.']) {
         for (let key in workspace.manifest.exports) {
-          let subExportsRequire = workspace.manifest.exports[key].require;
-          let subExportsImport = workspace.manifest.exports[key].import;
-          workspace.set(`exports["${key}"].require`, setExtension(subExportsRequire, cjsExt));
-          workspace.set(`exports["${key}"].import`, setExtension(subExportsImport, '.mjs'));
+          if (workspace.manifest.exports[key]) {
```

Comment:
Shouldn't these be checking if `require` or `import` exists? Currently it's the same check for both conditionals so they should be consolidated or aren't doing anything if that key always exists.